labcodes/confetti-storybook
# [CSYS-355] Migrating Alert to function component

Link to task:
[CSYS-355](https://labcodes.atlassian.net/browse/CSYS-355)

Staging app: https://CSYS-355-migrate-alert--confetti-storybook.netlify.app/

How to test:
After running your project locally, run the `Alert.test.js` to check if the Alert component is working as expected.

Description of your solution:

Removing previous mentions from Class Components and moving the Props outside the functional component scope.

Screenshots (when applicable):
![image](https://user-images.githubusercontent.com/12106738/184701011-03044e88-88f8-4ddd-9c5c-247e3128e3fc.png)
